profiles/arch/arm/armv5te/use.mask

```diff
@@ -1,2 +1,10 @@
 # Unmask armv5 specific USE flags
 -armv5te
+
+-cpu_flags_arm_v4
+-cpu_flags_arm_v5
+
+-cpu_flags_arm_edsp
+-cpu_flags_arm_thumb
+-cpu_flags_arm_vfp
+-cpu_flags_arm_vfp-d16
```

Comment:
I assume vfp means whatever the kernel means by vfp ;-). If you could figure that out, I'd appreciate.